README.md

```diff
@@ -43,13 +43,14 @@ Company                                           | Business/products
 [Envato](https://envato.com/)                     | Sites for creatives                      | Ruby, Web                                      | ✓                                    | Anywhere                | :keyboard: [Jobs](https://envato.com/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Envato-EI_IE700841.11,17.htm)
 [Eyeo](https://eyeo.com)                          | AdBlock Plus & More                      | Many                                           | ?                                    | Anywhere                | :keyboard: [Jobs](https://eyeo.com/jobs/)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-Eyeo-EI_IE848298.11,15.htm)
 [Fire Engine Red](http://fire-engine-red.com/)    | Education market solutions               | ?                                              | ?                                    | Anywhere                | :keyboard: [Jobs](http://fire-engine-red.com/about/#careers)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Fire-Engine-RED-EI_IE1339735.11,26.htm)
+[Gatsby](https://ghost.org/)                      | Platform for managing JAMstack websites  | Web, Node, React, TypeScript                   | ✓                                    | Anywhere                | :keyboard: [Jobs](https://www.gatsbyjs.com/careers/)<br>:open_hands: [Open Source](https://github.com/gatsbyjs/gatsby)
```

Comment:
Just a few comments:

* The link is to ghost.org rather than Gatsby :slightly_smiling_face: 
* Let's drop the open source link
* Is there a Glassdoor link?

Reply:
🤦‍♂️

Reply:
There was indeed a Glassdoor page! Thanks for pointing that out!